### DISCUSSION TITLE
Question about gonir

### DISCUSSION THREAD

**Discussion by tnmas** [nornir-automation/gornir#55] [General]
Hi,

I am using gonir for the first time and I was able to send some show commands to the nodes and got response too, but I have no idea how to configure those devices, is there any task(like netmiko_send_config for example) in gonir, which I can use to send configs to my devices?

Thank you in advance for the fast reply and your help!

**Comment by dbarrosop** [comment]
No, I am afraid there is no such a task included with gornir.
